plugins/chain_plugin/include/eosio/chain_plugin/blockvault_sync_strategy.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+#include <stdint.h>
+#include <string_view>
```

Comment:
Tested that it builds without including `#include <stdint.h>` and `#include <string_view>`